edgelet/Makefile

```diff
@@ -51,6 +51,8 @@ MAKE=make
 MKDIR_P=mkdir -p
 SED=sed
 HOST_URI_OPTION=
+CONFIG_YAML="config.yaml"
```

Comment:
Why do these two need to be variables? They can't have any other value anyway, so they don't need to be configurable.

Reply:
If your preference is not to have variables, I can certainly understand that. Reverted to filenames.